guides/common/modules/ref_supported-puppet-versions-and-system-requirements.adoc

```diff
@@ -0,0 +1,27 @@
+[id="supported-puppet-versions-and-system-requirements_{context}"]
+= Supported Puppet Versions And System Requirements
+
+Before you begin with the Puppet integration, review the supported Puppet versions and system requirements.
+
+Supported Puppet Versions::
+{Project} supports the following Puppet versions:
+ifndef::orcharhino[]
+* Puppet 7
+endif::[]
+ifndef::satellite[]
+* Puppet 6
+endif::[]
+
+System Requirements::
+Before you begin integrating Puppet with your {Project}, ensure that you meet the system requirements.
+For details, see
+ifndef::orcharhino[]
+https://puppet.com/docs/puppet/7/system_requirements.html[System Requirements for Puppet 7]
+ifndef::satellite[]
```

Comment:
```suggestion
ifdef::katello,foreman-el,foreman-deb[]
```

Right now, the `or` would show for orcharhino, even though it only links to docs for Puppet 6.

Reply:
Thanks! Fixed.